### ISSUE TITLE
[Docs] some problems in sample_data/README.md

### ISSUE THREAD

**Issue by xgdyp** [X-lab2017/open-digger#992]
![image](https://user-images.githubusercontent.com/37795442/194822096-8ab3efc1-8bbd-4153-a60c-432ccbf1143c.png)
In this document, I think we should use full container name like`docker-hub.x-lab.info/opendigger/open-digger-clickhouse-base:v1` instead of `open-digger-clickhouse-base:v1` to run command
```
docker run -d --name container_name -m 6G -p 8123:8123 -p 9000:9000 --ulimit nofile=262144:262144 --volume=%cd%/folder_name/:/data/ open-digger-clickhouse-base:v1
```

**Comment by xgdyp** [comment]
Thanks @Best-Eminem @zhicheng-ning @yoyo-wu98  for testing it.

**Comment by yoyo-wu98** [comment]
Thanks for your suggestion. SInce we've fixed this in #993 , I think we can close this issue.
